.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Thanks for trying out Zero to JupyterHub!
+
+If you're having issues with your deployment, it is
```

Comment:
Before filing an issue, please search issues to see if your question has been discussed before. If you need more information after searching, please feel free to message us on the gitter channel. Many JupyterHub community members watch the gitter channel so you will have the benefit of other users' experience as well as the JupyterHub team.

If you still wish to file an issue, please submit as much detail about your issue as possible. If you think it would be helpful, please include a scrubbed version of your `config.yaml` file. We've....